Sources/TensorFlow/Layers/Convolutional.swift

```diff
@@ -404,9 +404,9 @@ public struct TransposedConv2D<Scalar: TensorFlowFloatingPoint>: Layer {
     @differentiable
     public func callAsFunction(_ input: Tensor<Scalar>) -> Tensor<Scalar> {
         let batchSize = input.shape[0]
-        let w = (input.shape[1] - (1 * paddingIndex)) *
+        let w = (input.shape[1] - 1) *
           strides.0 + (filter.shape[0] * paddingIndex)
-        let h = (input.shape[2] - (1 * paddingIndex)) *
+        let h = (input.shape[2] - 1) *
           strides.1 + (filter.shape[1] * paddingIndex)
         let c = filter.shape[2]
         let newShape = Tensor<Int32>([Int32(batchSize), Int32(w), Int32(h), Int32(c)])
```

Comment:
It seems definitions of `w` and `h` are swapped.
(`input` has shape [batch, height, width, channels], right?)
They are swapped again in `newShape` creation. 
As a result, it's working correctly.

Reply:
@t-ae so are there any changes to be made wrt to these mistakes you pointed out?

Reply:
Now `w` means height and `h` means width.
Swapping these names would reduce confusion.